observability/otel/otel-collector/otel-collector-processors.adoc

```diff
@@ -578,3 +579,194 @@ config:
+                    },
+                    {
+                      name: composite-policy-3,
+                      type: always_sample
+                    }
+                  ],
+                  rate_allocation:
+                  [
+                    {
+                      policy: composite-rate-policy-1,
+                      percent: 50
+                    },
+                    {
+                      policy: composite-rate-policy-2,
+                      percent: 25
+                    }
+                  ]
+              }
+          },
+        ]
+# ...
+----
+<1> Wait time since the first span of a trace before making a sampling decision. Default is 30s.
+<2> Number of traces kept in memory. Default is 50000
+<3> Expected number of new traces (helps in allocating data structures). Default is 0.
+<4> Policy to sample all traces.
+<5> Policy to sample based on the duration of a trace. The duration is determined by looking at the earliest start time and latest end time, without taking into consideration what happened in between. Supplying no upper bound will result in a policy sampling anything greater than `threshold_ms`.
+<6> Policy to sample based on number attributes (resource and record).
+<7> Policy to sample a percentage of traces.
+<8> Policy to sample based upon the status code (OK, ERROR or UNSET).
+<9> Policy to sample based on string attributes (resource and record) value matches, both exact and regex value matches are supported.
+<10> Policy to sample based on the rate of spans per second.
+<11> Policy to sample based on the minimum and/or maximum number of spans, inclusive. If the sum of all spans in the trace is outside the range threshold, the trace will not be sampled.
+<12> Policy to sample based on TraceState value matches.
+<13> Policy to sample based on boolean attribute (resource and record).
+<14> Policy to sample based on given boolean OTTL condition (span and span event).
+<15> Policy to sample based on multiple policies, creates an AND policy.
+<16> Policy to drop (not sample) based on multiple policies, creates a DROP policy.
+<17> Policy to sample based on a combination of above samplers, with ordering and rate allocation per sampler. Rate allocation allocates certain percentages of spans per policy order. For example if we have set `max_total_spans_per_second` as `100` then we can set `rate_allocation` as follows: test-composite-policy-1 = 50 % of max_total_spans_per_second = 50 spans_per_second, test-composite-policy-2 = 25 % of max_total_spans_per_second = 25 spans_per_second, To ensure remaining capacity is filled use always_sample as one of the policies.
+
+
+Each policy will result in a decision, and the processor will evaluate them to make a final decision:
+
+* When there's a "drop" decision, the trace is not sampled;
+* When there's a "sample" decision, the trace is sampled;
+* In all other cases, the trace is NOT sampled
```

Comment:
@pavolloffay, are decisions to drop or sample individual traces anyhow visible to the user (like in a log or UI?) or is this, loosely speaking, a black box system?

Reply:
There are metrics exposed by the processor which can help to understand its behavior https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/tailsamplingprocessor#tracking-sampling-policy 

It would be maybe good to document them. Do you want me to update the PR or do you prefer to do it yourself?

Reply:
Thank you. Let me try. If it's too difficult for me, I'll ask for your help.